### PR TITLE
SIDM-3686 - add rule set for idam and x-forward-for

### PR DESCRIPTION
**Pull request by henrydobson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3686

### Change description ###

add rule set for idam and x-forward-for

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: modules/app-gateway-module/main.tf [hunk 1]
```diff
@@ -103,7 +104,8 @@ resource "azurerm_application_gateway" "ag" {
 
   dynamic "request_routing_rule" {
     for_each = [for app in var.frontends : {
-      name = app.name
+      name                  = app.name
```
Comment (timja) [inline modules/app-gateway-module/main.tf]: Can’t we just add it for everything? We don’t want special cases in here.

Comment (henrydobson) [reply modules/app-gateway-module/main.tf]: If that's ok @timja. I thought it would be preferred this way.

Comment (timja) [reply modules/app-gateway-module/main.tf]: this functionality is generic enough that it looks fine to add for everyone

**Comment by kremi** [comment]
> has this been tested in sandbox?

The rule was tested on Sandbox. The code has not been tested - what's a proper way of testing before merging?

**Comment by timja** [comment]
> > has this been tested in sandbox?
> 
> The rule was tested on Sandbox. The code has not been tested - what's a proper way of testing before merging?

running it locally with terraform in sandbox

**Comment by timja** [comment]
or comment the code that stops applies being run on PRs and allow it to run on sandbox

**Pull request merged**